### PR TITLE
Issue 10474 remove default repository

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

File: modules/core/core-api/src/main/java/com/enonic/xp/content/ContentConstants.java [hunk 1]
```diff
@@ -36,17 +36,18 @@ public final class ContentConstants
...
-    @Deprecated
-    public static final Repository CONTENT_REPO =
-        Repository.create().id( CONTENT_REPO_ID ).branches( Branches.from( BRANCH_DRAFT, BRANCH_MASTER ) ).build();
-
-    @Deprecated
-    public static final Context CONTEXT_DRAFT = ContextBuilder.create().branch( BRANCH_DRAFT ).repositoryId( CONTENT_REPO_ID ).build();
-
-    @Deprecated
-    public static final Context CONTEXT_MASTER = ContextBuilder.create().branch( BRANCH_MASTER ).repositoryId( CONTENT_REPO_ID ).build();
+//    @Deprecated
```
Comment (rymsha) [inline modules/core/core-api/src/main/java/com/enonic/xp/content/ContentConstants.java:39]: delete dead code

File: modules/core/core-app/src/main/java/com/enonic/xp/core/impl/app/ApplicationInfoServiceImpl.java [hunk 2]
```diff
@@ -103,7 +103,7 @@ public Descriptors<TaskDescriptor> getTaskDescriptors( final ApplicationKey appl
     @Override
     public Contents getContentReferences( final ApplicationKey applicationKey )
```
Comment (rymsha) [inline modules/core/core-app/src/main/java/com/enonic/xp/core/impl/app/ApplicationInfoServiceImpl.java:104]: `@Deprecated`

File: modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentServiceImpl.java [hunk 3]
```diff
@@ -813,14 +808,7 @@ public FindContentIdsByQueryResult find( final ContentQuery query )
     @Override
```
Comment (rymsha) [inline modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentServiceImpl.java:808]: `@Deprecated`

File: modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ProjectContentEventListener.java [hunk 4]
```diff
@@ -86,7 +86,8 @@ private void handleContentEvent( final Event event )
 
         if ( isContentEvent )
         {
-            this.simpleExecutor.execute( () -> doHandleContentEvent( nodes, event.getType() ) );
+            Context currentContext = ContextAccessor.current();
```
Comment (rymsha) [inline modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ProjectContentEventListener.java:89]: I think we need to copy context before giving it to executor

File: modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/current/GetCurrentSiteConfigHandler.java [hunk 5]
```diff
@@ -41,7 +40,7 @@ public PropertyTreeMapper execute()
             }
             else
             {
-                final Project project = this.projectService.get().get( ProjectName.from( ContextAccessor.current().getRepositoryId() ) );
+                final Project project = this.projectService.get().get( ProjectName.from( this.request.getRepositoryId() ) );
```
Comment (rymsha) [inline modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/current/GetCurrentSiteConfigHandler.java:43]: I would not do it in this PR. It is a tricky topic - where to get project from.
This change introduces ambiguity - project from request, but branch from context

File: modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ExceptionRendererImpl.java [hunk 6]
```diff
@@ -173,7 +173,7 @@ private PortalResponse doRenderCustomError( final PortalRequest req, final WebEx
         final Site siteInRequest = req.getSite();
         final Site site = siteInRequest != null
             ? siteInRequest
-            : callAsContentAdmin( () -> this.contentService.findNearestSiteByPath( req.getContentPath() ) );
+            : ( req.isSiteBase() ? callAsContentAdmin( () -> this.contentService.findNearestSiteByPath( req.getContentPath() ) ) : null );
```
Comment (rymsha) [inline modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ExceptionRendererImpl.java:176]: Looks strange - site is in request, bun not siteBase request?

**Review by rymsha — CHANGES_REQUESTED** [review]

**Review by rymsha — APPROVED** [review]

**Review by vbradnitski — APPROVED** [review]

**Pull request merged**